src/wlserver.cpp

```diff
@@ -1111,6 +1113,26 @@ void wlserver_touchmotion( double x, double y, int touch_id, uint32_t time )
 	bump_input_counter();
 }
 
+
+void wlserver_capture_touch_down( int touch_id )
+{
+	if ( wlserver_touch_down_ids.size() < WLSERVER_TOUCH_COUNT )
```

Comment:
I think we can get rid of `WLSERVER_TOUCH_COUNT` now that we no longer have a fixed sized array. Also we can drop `wlserver.touch_down`.